src/models/users.js

```diff
@@ -28,6 +28,10 @@ module.exports = (sequelize, DataTypes) => {
       password: {
         type: DataTypes.STRING,
         allowNull: false,
+      }, 
+      role: {
+        type: DataTypes.STRING,
+        allowNull: false,
```

Comment:
Maybe we should set default value to "user".

Reply:
Fixed